airbyte-server/src/main/java/io/airbyte/server/handlers/SchedulerHandler.java

```diff
@@ -116,7 +124,8 @@ public SchedulerHandler(final ConfigRepository configRepository,
                    final JsonSchemaValidator jsonSchemaValidator,
                    final JobPersistence jobPersistence,
                    final EventRunner eventRunner,
-                   final JobConverter jobConverter) {
+                   final JobConverter jobConverter,
+                   final ConnectionsHandler connectionsHandler) {
```

Comment:
Not for the scope of this review but to keep in mind for later: I think `discoverSchemaForSourceFromSourceId` should be move to the SourceHandler. We should get rid of the scheduler handler at some points (we don't have any scheduler anymore).